hail/src/main/c/NativeBoot.cpp

```diff
@@ -19,8 +19,8 @@ NATIVEMETHOD(jlong, NativeCode, dlopenGlobal)(
 }
 
 NATIVEMETHOD(jlong, NativeCode, dlclose)(
-  JNIEnv* env,
-  jobject /*thisJ*/,
+  JNIEnv*,
+  jobject,
```

Comment:
`gcc` doesn't like that `JNIEnv* env` is unused, so I remove it's name. Looks like that was already done for the other tow, but with commented name. I removed the comment. @catoverdrive, do you know why these methods have unused arguments?